mon-pix/translations/en.json

```diff
@@ -145,6 +145,80 @@
         },
         "placeholder": "Loading application"
       },
+      "feedback-panel": {
+        "actions": {
+          "open-close": "Notify a problem"
+        },
+        "description": "Pix is listening to your advices !*",
+        "form": {
+          "actions": {
+            "submit": "Submit"
+          },
+          "fields": {
+            "category-selection": {
+              "label": "I have difficulty with",
+              "options": {
+                "accessibility": "The challenge's accessibility",
+                "answer": "The answer",
+                "embed": "The embed",
+                "download": "The file to download",
+                "link": "The link",
+                "other": "Other",
+                "picture": "The picture",
+                "question": "The question",
+                "tutorial": "The tutorial"
+              }
+            },
+            "detail-selection": {
+              "label": "Add more details",
+              "options": {
+                "answer-not-accepted": "Ma answer is right but has not been accepted",
+                "answer-not-agreed": "I disagree with the answer",
+                "download": {
+                  "lost": {
+                    "label": "I can't find the downloaded file",
+                    "solution": "The file is save under “Téléchargements” ou “Downloads” by default. It could also be saved at the same place than your last download..."
```

Comment:
```suggestion
                    "solution": "The file is saved under “Téléchargements” or “Downloads” by default. It could also be saved at the same place than your last download..."
```